sdk/core/typespec_client_core/src/http/policies/retry/mod.rs

```diff
@@ -127,10 +137,10 @@ pub trait RetryPolicy: std::fmt::Debug + Send + Sync {
     }
 }
 
-/// The status codes where a retry should be attempted.
+/// Default status codes where a retry should be attempted.
 ///
 /// On all other 4xx and 5xx status codes no retry is attempted.
-const RETRY_STATUSES: &[StatusCode] = &[
+const DEFAULT_RETRY_STATUSES: &[StatusCode] = &[
```

Comment:
```suggestion
const DEFAULT_RETRY_STATUS_CODES: &[StatusCode] = &[
```

Reads better, and better aligns with `*status_codes` everywhere else.